spec/features/encounter_spec.rb

```diff
@@ -81,7 +81,7 @@
       end
     end
 
-    it "should only show sites with create encounter permission of context institution" do
+    xit "should only show sites with create encounter permission of context institution" do
```

Comment:
It is just that when you view a test order, Chris asked to remove the sample and test results data [everything below the patient data], and just wanted to make sure your team were happy with that before removing the tests.  Does it seem OK ? I can remove the tests if so.